apps/www/_blog/2023-08-07-hugging-face-supabase.mdx

```diff
@@ -0,0 +1,237 @@
+---
+title: 'Hugging Face is now supported in Supabase'
+description: We've added support Hugging Face support in our Python Vector Client and Edge Functions.
+launchweek: 8
+tags:
+  - launch-week
+  - announcements
+date: '2023-08-07'
+toc_depth: 3
+author: copple
```

Comment:
Is the author correct, should we add more people? @kiwicopple

Reply:
this should be Greg and Oli